include/spm/rel/sp4_13.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <common.h>
+#include <spm/evtmgr.h>
+#include <spm/npcdrv.h>
+#include <evt_cmd.h>
+
+CPP_WRAPPER(spm::sp4_13)
+
+char * returnCharAnimPos(char *param_1);
```

Comment:
Should probably have a map name prefix, and maybe a bit more of a specific name (for example, the function after it in ram could technically be called this too)